.circleci/config.yml

```diff
@@ -76,3 +76,42 @@ jobs:
       - deploy:
           command: |
             ./cc-test-reporter sum-coverage --output - --parts $CIRCLE_NODE_TOTAL coverage/codeclimate.*.json | ./cc-test-reporter upload-coverage --debug --input -
+  build-latest-container:
+    working_directory: ~/identity-pki
+    docker:
+      - image: circleci/ruby:2.6
+    steps:
+      - checkout
+      - setup_remote_docker
+      - run: |
+          rev=$(git rev-parse --short HEAD)
+          docker build -t logindotgov/pki:latest -t logindotgov/pki:"${rev}" -f Dockerfile .
+          echo $DOCKER_PASS | docker login -u $DOCKER_USER --password-stdin
+          docker push logindotgov/pki:"${rev}"
+          docker push logindotgov/pki:latest
+  build-release-container:
```

Comment:
So, `build-release-container` happens when there is a `CIRCLE_TAG`, which implies that we cut a release (?).
It seems like in that case this will do 2 container builds?

Reply:
Yes, that's the case. It creating the container because it's unable to share the context, but it will give it a different tag.

Reply:
does it make more sense to have the opposite of the `if` statement under `build-release-container` in the `build-latest-container`?  Or is it better for us to have both a new "latest" and a new "release" container when we cut a release?  I feel like I'm missing something here.